extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -2691,22 +2708,22 @@ static public function setFlags( $city_id, $city_flags, $skip=false, $reason = '
 	static public function getFlags( $city_id ) {
 		global $wgWikicitiesReadOnly;
 
-		if ( !self::isUsed() ) {
+		if ( !static::isUsed() ) {
 			Wikia::log( __METHOD__, 'info', 'WikiFactory is not used.' );
 			return false;
 		}
 
 		wfProfileIn( __METHOD__ );
 
-		$dbw = self::db( DB_SLAVE );
+		$dbw = static::db( DB_SLAVE );
 		$city_flags = $dbw->selectField(
 			'city_list',
 			'city_flags',
 			[ 'city_id' => $city_id ],
 			__METHOD__
 		);
 		//reduce log spam in wikifactory logs
-		//self::log( self::LOG_STATUS, sprintf('Binary flags %s read from city_flags', decbin( $city_flags ) ), $city_id );
+		//static::log( static::LOG_STATUS, sprintf('Binary flags %s read from city_flags', decbin( $city_flags ) ), $city_id );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)